docs/source/GETTING_STARTED.rst

```diff
@@ -0,0 +1,133 @@
+
+Cloud-Based Jumpbox Workflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Navigate to the `VM instances <https://console.cloud.google.com/compute/instances>`_ page via ``Compute Engine > VM instances``
+* Check that your boot disk is configured for ``Debian/Ubuntu 9`` operating system
+
+.. warning:: Container optimized images do not support kiosk installation.
+
+* All other settings can be left as defaults
+* After creating the instance, SSH into your instance either using the option provided by Google Cloud or through your local terminal.
+* If you have chosen to SSH into the machine from a terminal on your local machine, simply paste the following commands copied from the Docker installation guide for `Debian <https://docs.docker.com/install/linux/docker-ce/debian/>`_
+
+.. code-block:: bash
+
+    sudo apt-get update && \
+    sudo apt-get install -y apt-transport-https ca-certificates curl gnupg2 software-properties-common && \
+    curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add - && \
+    sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" && \
+    sudo apt-get update && \
+    sudo apt-get install -y containerd.io docker-ce docker-ce-cli git make vim
+
+* Alternatively, if you SSH'd in using Google Cloud's browser-based terminal, you will need to break that large clump of commands into individual commands (each semicolon denotes the end of a command) and paste them one at a time onto the command line. After this, you should see the kiosk GUI screen and can follow the kiosk configuration and launch process as usual.
+
+Starting the Kiosk
+^^^^^^^^^^^^^^^^^^
+
+You are now ready to start the kiosk! Those interested in Kiosk development should follow a different path to start the kiosk which is described in :ref:`DEVELOPER`.
+
+7. Start a terminal shell and install the DeepCell Kiosk wrapper script:
+
+.. code-block:: bash
+
+    docker run -e DOCKER_TAG=1.0.0 vanvalenlab/kiosk:1.0.0 | sudo bash
+
+.. note:: This command and the one that follows may need to be preceded by `sudo` depending on your permission settings.
+
+8. To start the kiosk, just run ``kiosk`` from the terminal shell
+
+.. todo::
+
+    Add screenshot of welcome screen
+
+DeepCell Kiosk Usage
+--------------------
+
+9. Once the Kiosk has started, select the configuration option for your chosen cloud provider (currently, only Google Cloud is supported) and fill out the configuration values as needed. Each response is pre-populated with a default value appropriate for most users (for more detailed information on each of these values refer to our [advanced documentation](docs/ADVANCED_DOCUMENTATION.md)). Once the Kiosk has been configured, you will be returned to the main menu and the word ``(active)`` will appear next to that cloud provider's configuration option.
+
+.. todo::
+
+    Write a complete description of each configuration option after the menu refactor is complete.
```

Comment:
I think this needs to be updated (possibly with screenshots) now that the menu has been refactored. Can I get some guidance on the major changes?

Reply:
Firstly, the advanced documentation link is broken.  In regard to the major changes, a bullet list is below:

- We no longer have multiple cloud providers, only GKE.
- Next screen will prompt for authentication with gcloud (if logged in previously it will ask if you want to continue with that account).
- Select your project from a drop-down of active projects.
- Enter your cluster name which must be no more than 18 characters.
- Enter your bucket name, which must be a unique value (this is where data is stored).
- Decide between "Default" and "Advanced" configuration. Default sets some basic values for region, node type, number of GPU nodes, etc.  Advanced lets you configure all of this individually.
- No more `(active)`. Once the cluster has been configured, you can create it. If you select create without configuring, it will warn you and send you back to the home screen.
- Create also gives you a confirmation page.